README.md

```diff
@@ -35,7 +35,8 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+I would use an Angular factory with this in Express:
+Instructor.findOneAndUpdate({name: req.params.name}, req.body.instructor, {new: true})
```

Comment:
Once you've found the instructor, how would you go about adding something to his/her `wishlist_items`?